mon-pix/app/components/navbar-header.js

```diff
@@ -1,9 +1,12 @@
-import { classNames } from '@ember-decorators/component';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+
+const FRENCH_DOMAIN_EXTENSION = 'fr';
 
-@classic
-@classNames('navbar-header')
 export default class NavbarHeader extends Component {
-  burger = null;
+  @service currentDomain;
+
+  get isFrenchDomainExtension() {
+    return FRENCH_DOMAIN_EXTENSION == this.currentDomain.getExtension();
```

Comment:
Pour la lecture, j'inverserais les deux : 
```suggestion
    return this.currentDomain.getExtension() === FRENCH_DOMAIN_EXTENSION;
```